src/shogun/base/AnyParameter.h

```diff
@@ -88,6 +88,16 @@ namespace shogun
 			return m_properties;
 		}
 
+		inline bool operator==(const AnyParameter& other)
+		{
+			return m_value == other.get_value();
```

Comment:
good? not good?

Reply:
Probably ok. Maybe needs a comment that it doesn't consider properties.